lib/dotenv_validator.rb

```diff
@@ -156,4 +166,46 @@ def self.root
       root_or_pwd
     end
   end
+
+  # Accepted options for the integer type
+  # @return [Array]
+  def self.integer_options
+    %w[int integer Integer]
+  end
+
+  # Accepted options for the float type
+  # @return [Array]
+  def self.float_options
+    %w[float Float]
+  end
+
+  # Accepted options for the string type
+  # @return [Array]
+  def self.string_options
+    %w[str string String]
+  end
+
+  # Accepted options for the email type
+  # @return [Array]
+  def self.email_options
+    %w[email Email]
+  end
+
+  # Accepted options for the url type
+  # @return [Array]
+  def self.url_options
+    %w[url Url]
```

Comment:
I think `Url` is not a standarrd Ruby type, maybe `URI`? but URI is too generic (we actually use the URI module to get the regexp for Email and Url validations)

Maybe we can just leave `email` and `url` for those 2 formats